src/funding/card/config.jsx

```diff
@@ -99,7 +99,23 @@ export function getCardConfig() : FundingSourceConfig {
 
         handleClick: true,
 
-        Logo: ({ fundingEligibility, locale, onClick }) => {
+        Logo: ({ fundingEligibility, locale, onClick, cardButtonExperiment }) => {
+            // Testing for Inline Guest black button
+            if (cardButtonExperiment && locale.lang === 'en' && locale.country === 'US') {
+                return (
+                    <button
+                        type="button"
+                        onClick={ event => onClick(event) }
+                        style="background: #000; color: #fff; font-size: 1.15em; border-radius: 4px !important;"
```

Comment:
Can we use a style tag and pass in nonce. Otherwise we'll hit csp errors

Reply:
do you mean adding the `style` tag right here?
```
<div>
<style nonce={nonce}>
...
</style>
<button ... />
<div>
```

Reply:
gotcha.